packages/react/src/composite/ScrollPane/CHANGELOG.md

```
@@ -0,0 +1,18 @@
+# Changelog
+
+The ScrollPane component handles responsive scrolls in lists that are overflowing inside a container.
+
+## 2025-07-28 `1.0.0`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)